server/config.js

```diff
@@ -20,4 +19,12 @@ global.NAP.Config = {
   // Security
   cookie_secret: process.env.COOKIE_SECRET || 'foo',
   jwt_secret: process.env.JWT_SECRET || 'foo',
-}
\ No newline at end of file
+  
+  passportEnabled: int(process.env.PASSPORT_DISABLED || '0') === 0,
+  graphqlEnabled: true,
+  graphqliqlEnabled: true,
```

Comment:
`GraphiQL` should enable by default in dev mode and opt-in at prod please use `graphiql_enabled` and respect `process.env.GRAPHIQL_ENABLED`